firedrake/interpolation.py

```diff
@@ -369,7 +379,7 @@ def _interpolate(self, *args, **kwargs):
         """
         pass
 
-    def assemble(self, tensor=None, default_missing_val=None):
+    def assemble(self, tensor=None, **kwargs):
```

Comment:
Please give this the full docstring treatment. This is a public method and it is not clear what is happening to kwargs.